src/components/Breadcrumbs/Breadcrumbs.module.css

```diff
@@ -94,6 +93,10 @@
  * Breadcrumbs link.
  */
 .breadcrumbs__link {
+  /* preset: body-xs */
```

Comment:
Note for reviewers: for all cases where we have a `font:` in a stylesheet, there should be a prefix comment saying which preset it maps to